test/e2e/network_test.go

```diff
@@ -0,0 +1,81 @@
+package e2e
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/openshift/windows-machine-config-operator/pkg/controller/retry"
+	wmc "github.com/openshift/windows-machine-config-operator/pkg/controller/windowsmachineconfig"
+	"github.com/openshift/windows-machine-config-operator/pkg/controller/windowsmachineconfig/nodeconfig"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	// hybridOverlayMac is an annotation applied by the hybrid-overlay
+	hybridOverlayMac = "k8s.ovn.org/hybrid-overlay-distributed-router-gateway-mac"
+)
+
+// testNetwork runs all the cluster and node network tests
+func testNetwork(t *testing.T) {
+	t.Run("Hybrid overlay annotations", testHybridOverlayAnnotations)
+	t.Run("OpenShift HNS networks", testHNSNetworksCreated)
+}
+
+// testHybridOverlayAnnotations tests that the correct annotations have been added to the bootstrapped node
+func testHybridOverlayAnnotations(t *testing.T) {
```

Comment:
nit: I wonder if combining these 2 tests into a single test would make sense so as to share the node objects or else we can also move nodes to globalContext just like how we moved windowsVMs.

Reply:
I do not want to share the node objects as they could be stale.

Reply:
Ok, to be fair, the node objects should be same across the tests suite as the nodes are created in `TestCreation` and rest of the test suite is sequential. I don't have any strong opinion though.

Reply:
Sure the nodes are created in `TestCreation` but there is a window where one of the annotations need not be present. I also don't want to  keep adding things to global context unless they are really required.

Reply:
> but there is a window where one of the annotations need not be present.

You mean the mac address annotation?